core/domain/exp_services.py

```diff
@@ -385,14 +520,28 @@ def get_recently_published_exp_summaries(limit):
 def get_all_exploration_summaries():
     """Returns a dict with all exploration summary domain objects,
     keyed by their id.
+
+    Returns:
+        dict. A dict with all ExplorationSummary domain objects keyed by their
+        exploration id.
     """
     return _get_exploration_summaries_from_models(
         exp_models.ExpSummaryModel.get_all())
 
 
 # Methods for exporting states and explorations to other formats.
 def export_to_zip_file(exploration_id, version=None):
-    """Returns a ZIP archive of the exploration."""
+    """Returns a ZIP archive of the exploration.
+
+    Args:
+        exploration_id: str. The id of the exploration to export.
+        version: int or None. If provided, this indicates which version of
+            the exploration to export. Otherwise, the latest version of the
+            exploration is exported.
+
+    Returns:
+        zipfile. A ZIP archive of the exploration.
```

Comment:
```
LOG core.domain.exp_services_test.ZipFileExportUnitTests.test_export_to_zip_file:
<type 'str'>
test_export_to_zip_file (core.domain.exp_services_test.ZipFileExportUnitTests)
```
So overall this returns '**str**' which is then converted into zip via _zipfile.ZipFile(StringIO.StringIO(zip_file_output))_ which is a zip file. Learnt a lot :) Thanks.